### PR TITLE
[SDXL-IP2P] Update README_sdxl, Replace the link for wandb log with the correct run

### PR DESCRIPTION
**Pull request by priprapre**
## What does this PR do?

It replaces the wandb run with the correct link in `README_sdxl.md` for instructpix2pix training. The new link for wandb run is from the [diffusers/sdxl-instructpix2pix-768](https://huggingface.co/diffusers/sdxl-instructpix2pix-768) model page. (The old one was pointing to SD1.5 instructpix2pix run)
## 
I think there is a typo on the model page as well. It says that batch size per GPU is 8 (x4 gradient accumulation=32) but in the wandb run, I see that it's 4 (x4 gradient accumulation=16).

Besides, I observe that some parameters differ from the script on `README_sdxl`. For example, the `diffusers/sdxl-instructpix2pix-768` model is trained with `learning_rate=5e-06` and `conditioning_dropout_prob=0.1` whereas the script defaults are `5e-05` and `0.05`. Hyperparameters differ from [SD1.5 IP2P training](wandb.ai/sayakpaul/instruct-pix2pix/runs/ctr3kovq/overview), so it might be useful to change those defaults for SDXL so that we would know which set of hyperparameters worked out well for SDXL case.
Maybe @harutatsuakiyama also has some insights here about the hyperparams working the best.

cc: @sayakpaul

### CODE REVIEW

**Review by sayakpaul — APPROVED** [review]

**Comment by sayakpaul** [comment]
Thanks!
